Frameworks/UIKit/UIScrollView.mm

```diff
@@ -217,7 +217,8 @@ - (BOOL)_isAnimating {
 
 - (void)_setupViewChangingHandler {
     __weak UIScrollView* weakself = self;
-    _viewChangingEventToken = _scrollViewer.ViewChanging(objcwinrt::callback([weakself] (const WF::IInspectable&, const Controls::ScrollViewerViewChangingEventArgs& arg) {
+    _viewChangingEventToken = _scrollViewer.ViewChanging(objcwinrt::callback([weakself](
+        const WF::IInspectable&, const Controls::ScrollViewerViewChangingEventArgs& arg) {
```

Comment:
Sorry, this was done when I "git commit"ed the change.

---
In reply to: [112262947](https://github.com/Microsoft/WinObjC/pull/2565#discussion_r112262947) [](ancestors = 112262947,112261921)